aspnetcore/migration/70-80.md

```diff
@@ -438,15 +438,25 @@ Updated configuration guidance appears in the following locations:
 
 If you previously followed the guidance in <xref:migration/inc/blazor?view=aspnetcore-7.0&preserve-view=true> for migrating a Blazor Server app with Yarp to .NET 6 or .NET 7, you can reverse the workaround steps that you took when following the article's guidance. Routing and deep linking for Blazor Server with Yarp work correctly in .NET 8.
 
-## Update Docker images
+## Docker
 
-For apps using Docker, update your *Dockerfile* `FROM` statements and scripts. Use a base image that includes the ASP.NET Core 8.0 runtime. Consider the following `docker pull` command difference between ASP.NET Core 7.0 and 8.0:
+### Update Docker images
+
+For apps using Docker, update the *Dockerfile* `FROM` statements and scripts. Use a base image that includes the ASP.NET Core 8.0 runtime. Consider the following `docker pull` command difference between ASP.NET Core 7.0 and 8.0:
 
 ```diff
 - docker pull mcr.microsoft.com/dotnet/aspnet:7.0
 + docker pull mcr.microsoft.com/dotnet/aspnet:8.0
 ```
 
+### Update Docker port
+
+The default ASP.NET Core port configured in .NET container images has been updated from port 80 to 8080.
+
+The new `ASPNETCORE_HTTP_PORTS` environment variable was added as a simpler alternative to `ASPNETCORE_URLS`.
```

Comment:
You could link to https://learn.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/endpoints?view=aspnetcore-8.0#specify-ports-only for more information.